pkg/resource/testdata/TestStatefulSetBuilder/default_secure.golden

```diff
@@ -78,35 +78,66 @@ spec:
         - mountPath: /cockroach/cockroach-data/
           name: datadir
         - mountPath: /cockroach/cockroach-certs/
-          name: certs
+          name: emptydir
+      securityContext:
+        fsGroup: 10001
+        runAsUser: 10001
       serviceAccountName: cockroach-database-sa
       terminationGracePeriodSeconds: 60
       volumes:
       - name: datadir
         persistentVolumeClaim:
           claimName: ""
+      - emptyDir: {}
+        name: emptydir
       - name: certs
         projected:
-          defaultMode: 256
+          defaultMode: 420
           sources:
           - secret:
               items:
               - key: ca.crt
+                mode: 504
                 path: ca.crt
               - key: tls.crt
+                mode: 504
                 path: node.crt
               - key: tls.key
+                mode: 504
                 path: node.key
               name: test-cluster-node
           - secret:
               items:
               - key: tls.crt
+                mode: 504
                 path: client.root.crt
               - key: tls.key
+                mode: 504
                 path: client.root.key
               name: test-cluster-root
+      initContainers:
+        - command:
+          - /bin/sh
+          - -c
+          - '>- cp -p /cockroach/cockroach-certs-prestage/..data/* /cockroach/cockroach-certs/
+            && chmod 700 /cockroach/cockroach-certs/*.key && chown 10001:10001 /cockroach/cockroach-certs/*.key'
+          image: cockroachdb/cockroach:v19.2.6
```

Comment:
this isn't a versioned test, so this image needs to be updated to v.20.2.x please